developer/decision-records/2024-05-20-api-versioning-2/README.md

```diff
@@ -0,0 +1,119 @@
+
+## Approach
+
+### Grouping of APIs
+
+This section outlines the different groups (or "contexts") of APIs that currently exist in the Connector project, with
+the notable exclusions of DSP.
+
+1. the `"management"` context: contains all endpoints that allow manipulating the business objects of EDC, such as
+   Assets, Policies etc. This API is not intended for internet-facing deployments, and
+   requires [additional protection](../../best-practices.md#11-exposing-apis-to-the-internet).
+2. the `"observability"` context: contains the Observability endpoints that are intended to track the application health
+   in containerized deployments, similar
+   to [Kubernetes readiness probes](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/).
+   This API should not be exposed outside the container/pod. This is currently located under the `"default"` context.
+3. the `"control"` context: contains endpoints that are designed for component-to-component interaction, for
+   example [Data Plane Signaling](https://github.com/eclipse-edc/Connector/tree/main/docs/developer/decision-records/2023-12-12-dataplane-signaling)
+   between control plane and data plane.
+4. the `"public"` context: this is the public-facing endpoint(s) exposed by the dataplane for HTTP-PULL transfers.
+5. the `"sts"` context: the endpoints that are exposed by the Secure Token Service
+6. the `"version"` context: contains the endpoint to access version information,
+   see [below](#accessing-versioning-information) for details.
+
+Every API group (or "context") has its own version, but all of them adhere to the same versioning scheme: all endpoints
+of one particular group are reachable using the same base path, including the version segment (
+e.g. `/api/management/v4/...`).
+As stated in
+the [predecessor D-R](https://github.com/eclipse-edc/Connector/tree/main/docs/developer/decision-records/2023-11-09-api-versioning),
+URL path only contains major versions, prefixed with a "v", e.g. `"/v3", "/v4".`
+
+_Note that although this decision record only mentions APIs of the connector, the assertions and statements hold true for
+other components as well. APIs of other components, even if their semantics are similar (e.g. "IdentityHub Management API"), are
+technically treated as separate groups/contexts._
+
+### Accessing versioning information
+
+A new `"version"` context will be introduced that contains the Version Information API. The purpose of this API is to
+provide version information about each API that is deployed in the current runtime. With a simple `GET` request users
+can obtain that information:
+
+```shell
+GET /api/version
+```
+
+returns a response like the following:
+
+```json
+{
+   "management": [
+      {
+         "version": "3.0.0",
+         "urlPath": "/v3",
+         "last_updated": "2024-05-20T14:02:41Z"
```

Comment:
```suggestion
         "lastUpdated": "2024-05-20T14:02:41Z"
```

plus, a description on what `lastUpdated` represent needs to be explicit (I'm not sure, it refers to the commit timestamp? release timestamp? manual updated timestamp?)

Reply:
can't be the commit timestamp. IMO this has to be a "manually" entered timestamp. I'm putting this in quotes, because "manual" could also mean "with a script".